### PR TITLE
Update release notes for oneDPL 2021.7.1

### PR DESCRIPTION
**Pull request by ValentinaKats**
Signed-off-by: Valentina Kats valentina.kats@intel.com

### CODE REVIEW

File: documentation/release_notes.rst [hunk 1]
```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
 ===============
 
-Deprecation Notice
-------------------
-- Deprecated support of C++11 for Parallel API with host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``).
-  C++17 is the minimal required version going forward.
+New Features
+------------
+- Added namespace alias ``dpl = oneapi::dpl``.
+- Added a function that constructs a zip_iterator using a std::tuple of iterators.
```
Comment (akukanov) [inline documentation/release_notes.rst:17]: Maybe better "added possibility to construct a zip_iterator out of a std::tuple of iterators"?
@timmiesmith Is there a specification update patch for this?

File: documentation/release_notes.rst [hunk 2]
```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+- Added 9 more serial-based versions of algorithms: ``is_heap``, ``is_heap_until``, ``make_heap``, ``push_heap``, ``pop_heap``, ``is_sorted``, ``is_sorted_until``, ``partial_sort``, ``partial_sort_copy``. Please refer to `Tested Standard C++ API Reference https://github.com/oneapi-src/oneDPL/blob/main/documentation/library_guide/tested_standard_cpp_api.rst#tested-standard-c-api-reference`.
   
 Fixed Issues
```
Comment (akukanov) [inline documentation/release_notes.rst:20]: @MikeDvorskiy should your fix for in-place scan (https://github.com/oneapi-src/oneDPL/commit/7595b51cdee2044ae4cc9454bbcb1ef874c7dad3) be mentioned in release notes?

Comment (ValentinaKats) [reply documentation/release_notes.rst:20]: Fix for in-place scan is still work in progress

File: documentation/release_notes.rst [hunk 3]
```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+New in 2021.7.1
 ===============
 
-Deprecation Notice
-------------------
-- Deprecated support of C++11 for Parallel API with host execution policies (``seq``, ``unseq``, ``par``, ``par_unseq``).
-  C++17 is the minimal required version going forward.
+New Features
+------------
+- Added namespace alias ``dpl = oneapi::dpl``.
```
[the remark this answers is not in the document]
Comment (ValentinaKats) [reply documentation/release_notes.rst]: Moved to fixed issues section

File: documentation/release_notes.rst (hunk 1 above)
Comment (ValentinaKats) [reply documentation/release_notes.rst]: Applied suggestion

File: documentation/release_notes.rst [hunk 4]
```diff
@@ -8,26 +8,23 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
...
+New Features
+------------
+- Added possibility to construct a zip_iterator out of a std::tuple of iterators.
+- Added 9 more serial-based versions of algorithms: ``is_heap``, ``is_heap_until``, ``make_heap``, ``push_heap``, ``pop_heap``, ``is_sorted``, ``is_sorted_until``, ``partial_sort``, ``partial_sort_copy``. Please refer to `Tested Standard C++ API Reference https://github.com/oneapi-src/oneDPL/blob/main/documentation/library_guide/tested_standard_cpp_api.rst#tested-standard-c-api-reference`.
   
 Fixed Issues
 ------------
-- Fixed a kernel name definition error in range-based algorithms and ``reduce_by_segment`` used with
-  a device_policy object that has no explicit kernel name.
+- Added namespace alias ``dpl = oneapi::dpl``.
```
Comment (akukanov) [inline documentation/release_notes.rst:21]: ... into all public headers.

Comment (ValentinaKats) [reply documentation/release_notes.rst]: Applied suggestion

**Review by akukanov — APPROVED** [review]

**Pull request merged**